src/main/java/com/kodekonveyor/repo/api/SumtiEntityRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.kodekonveyor.repo.api;
+
+import java.util.Optional;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface SumtiEntityRepository
+    extends CrudRepository<SumtiEntity, Long> {
+
+  Optional<SumtiEntity> findByLerpoi(LerpoiEntity lerpoiEntity);
```

Comment:
One Lerpoi can belong to more sumtis, therefore you need to return List here instead of Optional.